lib/v8.js

```diff
@@ -42,3 +42,7 @@ exports.getHeapStatistics = function() {
 };
 
 exports.setFlagsFromString = v8binding.setFlagsFromString;
+
+exports.getHeapSpaceStatistics = function() {
```

Comment:
For reference: This comment is out of date and was made when the code looked this way:

```
exports.getHeapSpaceStatistics = function() {
  return v8binding.getHeapSpaceStatistics();
};
```